packages/components/src/sortTabs/SortTab.js

```diff
@@ -0,0 +1,169 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import {
+  Text,
+  StyleSheet,
+  Icon,
+  Hoverable,
+  designTokens,
+} from '@kiwicom/universal-components';
+
+import TouchableWithoutFeedback from '../TouchableWithoutFeedback';
+import type { SortIconTypes, SortTypes, PriceDurationInfo } from './SortTabs';
+
+export type Props = {|
+  +label: string,
+  +value: SortTypes,
+  +isFirst: boolean,
+  +isLast: boolean,
+  +isActive: boolean,
+  +priceDurationInfo: ?PriceDurationInfo,
+  +icon: SortIconTypes,
+  +onPress: SortTypes => void,
+|};
+
+type State = {|
+  isHovered: boolean,
+|};
+
+export default class SortTab extends React.Component<Props, State> {
+  state = {
+    isHovered: false,
+  };
+
+  handleOnMouseEnter = () => {
+    this.setState({ isHovered: true });
+  };
+
+  handleOnMouseLeave = () => {
+    this.setState({ isHovered: false });
+  };
+
+  handleOnPress = () => {
+    const { value, onPress } = this.props;
+    onPress(value);
+  };
+
+  render() {
+    const {
+      label,
+      isFirst,
+      isLast,
+      isActive,
+      icon,
+      priceDurationInfo,
+    } = this.props;
+    const { isHovered } = this.state;
+    return (
+      <Hoverable
+        onMouseEnter={this.handleOnMouseEnter}
+        onMouseLeave={this.handleOnMouseLeave}
+      >
+        <TouchableWithoutFeedback onPress={this.handleOnPress}>
+          <View
+            style={[
+              styles.container,
+              isFirst && styles.containerFirst,
+              isLast && styles.containerLast,
+              isActive && styles.containerActive,
+              isHovered && styles.containerHovered,
+            ]}
+          >
+            <View>
+              <Text
+                style={[styles.label, isActive && styles.labelActive]}
+                align="left"
+                numberOfLines={1}
+              >
+                {label}
+              </Text>
+              <View style={styles.priceDurationContainer}>
+                <Text style={styles.label} align="left" numberOfLines={1}>
```

Comment:
I don't know maybe I am too conservative but when there is no data for display I would expect not to render empty container and text or whatever if it not necessary. 

What do you think? @jvaclavik @JoseDuSV

Reply:
I agree Filip, I also thought of this but added lots of conditions. I think it's better not to render the empty container :D will change it.

Reply:
For me it depends on situation. Sometimes it's ok to keep component there if it's for example somehow affecting rest of the layout. But in this case using conditions makes more sense to me.